src/test/kotlin/app/batch/HttpWriterTest.kt

```diff
@@ -99,7 +99,7 @@ class HttpWriterTest {
         val formattedMessages = logCaptor.allValues.map { it.formattedMessage }
         assertEquals("""Writing items to S3", "number_of_items":"1"""", formattedMessages[0])
         assertEquals("""Checking item to  write", "file_name":"db.core.addressDeclaration-000001.txt.bz2.enc", "full_path":"exporter-output\/job01\/db.core.addressDeclaration-000001.txt.bz2.enc"""", formattedMessages[1])
-        assertEquals("""Found collection of file name", "collection":"db.core.addressDeclaration", "file_name":"exporter-output\/job01\/db.core.addressDeclaration-000001.txt.bz2.enc"""", formattedMessages[2])
+        assertEquals("""Found collection of file name", "collection":"addressDeclaration", "file_name":"exporter-output\/job01\/db.core.addressDeclaration-000001.txt.bz2.enc"""", formattedMessages[2])
```

Comment:
if we remove `db.core` are we specifying the database somewhere else?

Reply:
Yes in the database header.